docs/index.md

```diff
@@ -248,6 +248,37 @@ To facilitate this workflow, the issuer can include a link to help the user down
   ]
 }
 ```
+#### on Android
+When transferring a SMART Health Card from one app to another on Android, use:
+
+```java
+Intent intent = new Intent();
+intent.setAction(Intent.ACTION_SEND);
+intent.setType("application/smart-health-card");
+intent.putExtra(Intent.EXTRA_STREAM, uri);
+intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+startActivityForResult(Intent.createChooser(intent, ...
+```
+
+To receive a SMART Health Card in an Android app, register an intent filter, like:
+```xml
+    <intent-filter>
+        <action android:name="android.intent.action.SEND"/>
+        <category android:name="android.intent.category.DEFAULT"/>
+        <data android:mimeType="application/smart-health-card"/>
+    </intent-filter>
+
+<!-- other apps on the device (e.g. Gmail, Files) may use ACTION_VIEW and a binary MIME type, to be safe also support the below -->
```

Comment:
If apps need to support `ACTION_VIEW` with this MIME type for full compatibility, why don't we just specify this (instead of `ACTION_SEND`) above? Can we describe the advantages of `SEND`?

Reply:
> would also like to support opening a SHC from the Files app & other apps.

If I understand correctly, there are some apps (like GMail) where clicking an SHC attachment file triggers `ACTION_SEND`, and others apps (like Files) where clicking an SHC file in a local directory tree triggers `ACTION_VIEW`, even though these "feel" pretty equivalent to an end-user? If that's the case, it makes sense to recommend that receivers register for both, sure.

Reply:
Yes, this explanation sounds right from my experience. `ACTION_SEND` vs `ACTION_VIEW` seems to be more of a "product" decision, so perhaps we can simply identify the differences between the two, include sample implementations, and let implementers choose for themselves. How does that sound to folks?